src/Gui/Workbench.cpp

```diff
@@ -812,12 +813,21 @@ ToolBarItem* StdWorkbench::setupToolBars() const
     // View
     auto view = new ToolBarItem( root );
     view->setCommand("View");
-    *view << "Std_ViewFitAll" << "Std_ViewFitSelection" << "Std_ViewIsometric"
-          << "Std_ViewFront"<< "Std_ViewTop" << "Std_ViewRight"
-          << "Std_ViewRear" << "Std_ViewBottom"<< "Std_ViewLeft"
+    *view << "Std_ViewFitAll" << "Std_ViewFitSelection" << "Std_ViewGroup"
           << "Separator" << "Std_DrawStyle" << "Std_TreeViewActions"
           << "Separator" << "Std_MeasureDistance";
 
+    // Individual views
+    auto individualViews = new ToolBarItem(root, ToolBarItem::DefaultVisibility::Hidden);
+    individualViews->setCommand("Individual views");
+    *individualViews << "Std_ViewIsometric"
+                     << "Std_ViewFront"
+                     << "Std_ViewRight"
+                     << "Std_ViewTop"

```

Comment:
I have been asked in the past to re-add the separator in the middle of the standard view. It makes it easier on the eye. So you probably want to add the separator in the middle of this toolbar

Reply:
it makes not much sense to have a separator between the top and the right view IMO. @obelisk79 what is your take on that?
current:
![grafik](https://github.com/FreeCAD/FreeCAD/assets/6246609/355f82a5-9f98-4c55-9534-1a20f25bd225)

PR:
![grafik](https://github.com/FreeCAD/FreeCAD/assets/6246609/8f58c692-fb47-47c1-9fb2-4836640a1e22)


There is also no Separator in the right click menu or the toolbar currently!

Reply:
There is no separator in the toolbar because I removed them when I reworked the toolbar 😄 
Since someone asked that we put them back. So I'm mentioning it.